scripts/2010/01/report_errors.py

```diff
@@ -5,7 +5,10 @@
 from optparse import OptionParser
```

Comment:
Is this file used?

Reply:
https://github.com/internetarchive/olsystem/blob/master/etc/cron.d/openlibrary.allnodes#L22